day_6/exercises/burrito.js

```diff
@@ -15,12 +15,28 @@ class Burrito {
     this.protein = protein;
     this.base = base;
     this.toppings = toppings;
-  }
+}
 };
 
+
 var dinner = new Burrito("Beans", "Rice", ["cheese", "salsa", "guacamole"]);
 
-// Call the methods here
+
+function addTopping() {
+dinner.toppings.push("sour cream");
```

Comment:
@holladayian how would you add methods to the Burrito class, so that these are defined for all instances of a burrito?